src/gui/qgsmetadatawidget.cpp

```diff
@@ -477,7 +481,14 @@ void QgsMetadataWidget::saveMetadata( QgsLayerMetadata &layerMetadata ) const
   layerMetadata.setConstraints( constraints );
 
   // CRS
-  layerMetadata.setCrs( selectionCrs->crs() );
+  if ( selectionCrs->crs().isValid() )
```

Comment:
This change doesn't actually do anything - if selectionCrs->crs() is invalid, then it's the same as QgsCoordinateReferenceSystem()

Reply:
Oh ok, I didn't know/check that `selectionCrs->crs()` will return an null CRS if it was invalid. Thanks